protocols/dcutr/src/behaviour.rs

```diff
@@ -316,6 +316,19 @@ impl NetworkBehaviour for Behaviour {
 
         Poll::Pending
     }
+
+    fn on_swarm_event(&mut self, event: FromSwarm<Self::ConnectionHandler>) {
+        match event {
+            FromSwarm::ConnectionEstablished(connection_established) => {
+                self.on_connection_established(connection_established)
+            }
+            FromSwarm::ConnectionClosed(connection_closed) => {
+                self.on_connection_closed(connection_closed)
+            }
+            FromSwarm::DialFailure(dial_failure) => self.on_dial_failure(dial_failure),
+            _ => {}
```

Comment:
I would be in favor of not using a wildcard (i.e. `_`) here. In case we ever introduce a new `FromSwarm` event variant, I would appreciate the compiler error message. Instead I think we should write out the events here and have the match arm be a simple `{}`. I don't think that would be too verbose.

What do folks think?

Reply:
Just saw that you introduced the `swarm_event_variants`. I would prefer the my above suggestion. I don't think it is too verbose. I trust the compiler more than my regex-search skills across the repository.

I am guessing that you find this too verbose @thomaseizinger and @jxs?

Reply:
updated :)